src/main/java/pro/sky/skyprospringcoursework21/JavaQuestionService.java

```diff
@@ -5,19 +5,16 @@
 
 import java.util.*;
 
-@Service
-@Qualifier("java")
+@Service("javaQuestionService")
+//@Qualifier("java")
 public class JavaQuestionService implements QuestionService{
-    private final JavaQuestionRepository javaQuestionRepository = new JavaQuestionRepository();
-
-
-
+    private final JavaQuestionRepository javaQuestionRepository;
```

Comment:
```suggestion
    private final QuestionRepository javaQuestionRepository;
```
а в конструкторе Qualifier